tests/test_admin.py

```diff
@@ -0,0 +1,28 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase, Client
+from django.urls import reverse
+
+
+class AdminTest(TestCase):
+    def setUp(self) -> None:
+        self.client = Client()
+        self.admin_user = get_user_model().objects.create_superuser(
+            username="admin",
+            password="admin12345"
+        )
+        self.client.force_login(self.admin_user)
+        self.driver = get_user_model().objects.create_user(
+            username="user",
+            password="user12345",
+            license_number="ABC12345"
+        )
+
+    def test_driver_list_display(self):
+        url = reverse("admin:taxi_driver_changelist")
+        res = self.client.get(url)
+        self.assertContains(res, self.driver.license_number)
+
+    def test_driver_detailed_list_display(self):
```

Comment:
U are checking license number existence in this tests, it seems to me, that is better to rename this tests